typegraph/node/sdk/src/runtimes/deno.ts

```diff
@@ -38,6 +38,26 @@ export interface DenoImport {
   effect?: Effect;
 }
 
+function stringifyFn(code: string | Function) {
+  if (typeof code == "function") {
+    const source = code.toString();
+    const namedFnMatch = source.match(/function\s*(\*?\s*[a-zA-Z0-9_]+)/);
+    if (namedFnMatch) {
+      const [, name] = namedFnMatch;
+      if (name.replace(/s/g, "").startsWith("*")) {
```

Comment:
Do you mean character `'s'`? Or perhaps `\s`?

Reply:
Nice catch!